index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <header id="header">
+        <img src="#" alt="img" id="header-img">
+        <nav id="nav-bar">
+            <ul>
+                <li><a href="#video" class="nav-link">SectionA</a></li>
+                <li><a href="#form" class="nav-link">SectionB</a></li>
+                <li><a href="#email" class="nav-link">SectionC</a></li>
+            </ul>
+        </nav>
+        <div>
+            <iframe src="https://youtu.be/y8Yv4pnO7qc" id="video"></iframe>
+        </div>
+        <form action="https://www.freecodecamp.com/email-submit" id="form">
+            <input type="email" name="email" id="email" placeholder="Email" required>
```

Comment:
Iframe and form section should be outside the header tags inside the main section.